README.md

```diff
@@ -170,6 +170,18 @@ configured for the repository (see these [instructions](#prow-setup) for info on
      1. Use [kubeflow.testing.run_e2e_workflow](https://github.com/kubeflow/testing/tree/master/py/kubeflow/testing/run_e2e_workflow.py)
         to run the Argo workflow.
    * Add a `prow_config.yaml` file that will be passed into run_e2e_workflow to determine which ksonnet app to use for testing. An example can be seen [here](https://github.com/kubeflow/kubeflow/blob/master/prow_config.yaml).
+     * Workflows can optionally be scoped by job type (presubmit/postsubmit) or modified directories. For example:
+
+       ```
+       workflows:
+        - app_dir: kubeflow/testing/workflows
+          component: workflows
+          name: unittests
+          job_types: presubmit
+          include_dirs: foo/*,bar/*
```

Comment:
job_types and include_dirs should be arrays of strings e.g.

include_dirs:
  - foo/*
  - bar/*